src/main/kotlin/Main.kt

```diff
@@ -1,165 +1,38 @@
 val letters = arrayOf('A', 'B', 'C')
-val playedMoves = mutableListOf<String>()
-val boardArray = arrayOf(charArrayOf(' ', ' ', ' '), charArrayOf(' ', ' ', ' '), charArrayOf(' ', ' ', ' '))
+
 fun main() {
     var isPLaying = true
-
-    // starting game board doesn't need all the extra values
-    println("  1   2   3")
-    println("A   |   |   ")
-    println("  ---------")
-    println("B   |   |   ")
-    println("  ---------")
-    println("C   |   |   ")
+    val board = Board()
```

Comment:
Refactoring the board out makes this main function much smaller and focused on 'gui' behaviors. (technically tui, or terminal user interface)